File: Laboratorios C++/Nombre10VecesFor/Nombre10VecesFor.cpp
#include <iostream>
using namespace std;

int main(){
    //definición de variables
    string nombre;
    int i;

        //le pide al usuario que ingrese su nombre
        cout << "Ingrese su nombre: ";
        cin >> nombre;

    //la i se suma hasta 10
    for(i = 1; i <= 10; i++){
        //imprime el número de veces hasta donde sumó
        cout <<"Hola "<< nombre <<"!"<<endl;
    }
    return 0;
}

File: Laboratorios Java/Nombre10VecesFor/src/Nombre10VecesFor.java
import java.util.Scanner;

public class Nombre10VecesFor {
    public static void main(String[] args) throws Exception {
        Scanner scanner = new Scanner(System.in);

        //le pide al usuario que ingrese su nombre
        System.out.print("Igrese su nombre: ");

        //ell nombre ingresado se almacena
        String nombre = scanner.nextLine();

        //la i se suma hasta 10
        for(int i = 1; i <= 10; i++){

            //imprime el número de veces hasta donde sumó
            System.out.println("Hola "+ nombre +"!");
        }
        scanner.close();
    }
}